Makefile

```diff
@@ -4,6 +4,9 @@ DEFAULT_IMAGE_VERSION ?= $(BASE_BRANCH)
 export BASE_BRANCH
 export DEFAULT_IMAGE_VERSION
 
+DEFAULT_REPO = "quay.io/submariner"
+export DEFAULT_REPO
```

Comment:
This isn’t necessary since the variable is only used in the Makefile:

```suggestion
```

Reply:
That’s wrong, we’re using the shell variable (`$${…}`) so we do need to export it.